public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -87,7 +86,8 @@ export class SharedPreferences extends PureComponent<Props, State> {
     });
   }
 
-  onSubmitForm = async () => {
+  onSubmitForm = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
```

Comment:
Why do we need this prevent default here?

Reply:
Since we're not using `handleSubmit` here (it's not really necessary as we don't register form fields), the page will restart after the form is submitted, which we don't want in all the cases.